fabric-fluids-v1/src/main/java/net/fabricmc/fabric/access/fluids/BucketItemAccess.java

```diff
@@ -0,0 +1,7 @@
+package net.fabricmc.fabric.access.fluids;
```

Comment:
This should be in `impl`.

Reply:
wait really? why do duck interfaces go in impl?

Reply:
If it's not an API class or something with `@Mixin`, it should be in `impl`. I believe checkstyle rejects this sort of structuring as well.